I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/character1.rpy", line 4, in script
    "se ah escogido a [mc] como personaje principal"
KeyError: 'mc'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/character1.rpy", line 4, in script
    "se ah escogido a [mc] como personaje principal"
  File "C:\Users\Notsume\Documents\universidad\proyecto de titulo\sdk\renpy\ast.py", line 930, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\Notsume\Documents\universidad\proyecto de titulo\sdk\renpy\exports.py", line 1473, in say
    who(what, *args, **kwargs)
  File "C:\Users\Notsume\Documents\universidad\proyecto de titulo\sdk\renpy\character.py", line 1337, in __call__
    what = self.prefix_suffix("what", self.what_prefix, what, self.what_suffix)
  File "C:\Users\Notsume\Documents\universidad\proyecto de titulo\sdk\renpy\character.py", line 1251, in prefix_suffix
    return (sub(prefix) + sub(body) + sub(suffix))
  File "C:\Users\Notsume\Documents\universidad\proyecto de titulo\sdk\renpy\character.py", line 1231, in sub
    return renpy.substitutions.substitute(s, scope=scope, force=force, translate=translate)[0]
  File "C:\Users\Notsume\Documents\universidad\proyecto de titulo\sdk\renpy\substitutions.py", line 278, in substitute
    s = formatter.vformat(s, (), kwargs) # type: ignore
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/string.py", line 165, in vformat
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/string.py", line 205, in _vformat
  File "C:\Users\Notsume\Documents\universidad\proyecto de titulo\sdk\renpy\substitutions.py", line 168, in get_field
    obj, arg_used = super(Formatter, self).get_field(field_name, args, kwargs)
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/string.py", line 270, in get_field
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/string.py", line 227, in get_value
KeyError: 'mc'

Windows-10-10.0.19041 AMD64
Ren'Py 8.1.1.23060707
Transendencia 0.1
Tue Sep 26 17:12:11 2023
